packages/client/components/ActivityLibrary/ActivityGrid.tsx

```diff
@@ -31,6 +31,7 @@ const ActivityGrid = ({templates, selectedCategory}: ActivityGridProps) => {
               theme={CATEGORY_THEMES[template.category as CategoryID]}
               title={template.name}
               type={template.type}
+              templateRef={template}
```

Comment:
feels 2 separate components (one with require prop) would be good to avoid cases like this